lib/spack/spack/package_base.py

```diff
@@ -194,18 +194,29 @@ class WindowsRPathMeta(object):
     they would a genuine RPATH, i.e. adding directories that contain
     runtime library dependencies"""
 
-    def add_search_paths(self, *path):
+    def add_search_paths(self, *paths):
         """Add additional rpaths that are not implicitly included in the search
         scheme
         """
-        self.win_rpath.include_additional_link_paths(*path)
+        self.win_rpath.include_additional_link_paths(*paths)
+
+    def add_internal_links(self, *paths):
+        """Add additional rpaths internal to a package, i.e.
+        linking pkg.prefix.bin to pkg.prefix.lib.site-packages"""
```

Comment:
(this is only important if you disagree with [1])

I'm not sure the example is clarifying: I think it's sufficient to say that `*paths` designates a set of directories that contain libraries, and it is assumed that libraries in the package will need to be able to find the libraries in these paths.

Reply:
[1] works.